models/stg_amazon_ads__ad_group_level_report.sql

```diff
@@ -23,40 +23,40 @@ final as (
     select 
         _fivetran_synced,
         ad_group_id,
-        attributed_sales_same_sku_14_d,
-        attributed_sales_same_sku_1_d,
-        attributed_sales_same_sku_30_d,
-        attributed_sales_same_sku_7_d,
+        attributed_sales_same_sku_14_d, 
+        attributed_sales_same_sku_1_d, 
+        attributed_sales_same_sku_30_d, 
+        attributed_sales_same_sku_7_d, 
```

Comment:
Same comment from the macro. We don't want to include any non-standard metrics as default fields within the staging models. These should instead be passthrough via the variables you highlight in the README and can be [used](https://github.com/fivetran/dbt_apple_search_ads_source/blob/db9bb313686b4ea19e3c2b41e6df4cb62917862e/models/stg_apple_search_ads__campaign_report.sql#L33) in the staging model. 

Please remove any non-standard fields from the staging model. Be sure to also remove the definitions from the .yml if they are included there as well.

Reply:
You will want to make these updates for all other staging `_report` type models as well.

Reply:
done